Features3/tests/features/searchContact.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Search contact
+    As a user
+    I want to search contact by contact name
+    So that only searched contact is displayed in contact list
+
+    Background:
+        Given the User has logged in and navigated to Contact Fox page
+        And the following contacts has been created 
+            |name|email        |phone     |contact-type|
+            |abc |abc@gmail.com|9898989898|personal    |
+            |xyz |xyz@gmail.com|9876543210|personal    | 
+            |Ram |ram@gmail.com|9801234567|professional|
+            |Hari|hari@yahoo.com|9801234567|professional|
+
+    Scenario Outline: Search contact with contact name
+        When User search contact with contact name "<name>"
+        Then contact with name "<name>" should only be displayed in contact list
+        Examples:
+            |name|
+            |abc |
+            |xyz |
+            |Ram |
+            |Hari|
+
+    Scenario Outline: Search contact with string input
+        When User search contact with string input "<inputData>"
+        Then contacts which contains "<inputData>" in contact name or in email should be displayed in contact list
+        Examples:
+            |inputData|
+            |g        |
+            |y        |
+            |ram      |
+            |b        |
```

Comment:
this scenario is valid but need a good computation. Can you also add another scenario search by mail and then it will be good to go.